pkg/drivers/kic/oci/oci.go

```diff
@@ -156,12 +156,16 @@ func CreateContainerNode(p CreateParams) error {
 		if s != "running" {
 			return fmt.Errorf("temporary error created container %q is not running yet", p.Name)
 		}
+		if s == "running" {
```

Comment:
s has to be running at this point, so this if statement is not necessary.

Reply:
I was thinking to have that for logging